task/01-strings-tasks.js

```diff
@@ -221,7 +225,22 @@ function getRectangleString(width, height) {
  *
  */
 function encodeToRot13(str) {
-    throw new Error('Not implemented');
+    const shift = 13;
+    let encoded = '';
+    let chars = [...str];
+    for (let i = 0; i < chars.length; i++) {
+        let code = str.charCodeAt(i);
+
+        if (code >= 65 && code <= 90)
+            // lower case
+            code = ((code - 65) + shift) % 26 + 65;
+        else if (code >= 97 && code <= 122)
+            // upper case
+            code = ((code - 97) + shift) % 26 + 97;
+
+        encoded += String.fromCharCode(code);
+    }
```

Comment:
Quite cool! Most solutions just use 2 arrays and map one to another, but this one is looking not worse